### ISSUE TITLE
grab movie metadata for collections

### ISSUE THREAD

**Issue by CuddleBear92** [Darklyter/StashPlexAgent.bundle#14]
Currently only the collection itself is made but no data from the movies is pulled at all like the cover image or description which can be used by collections in Plex.

- [ ] Movie Cover Image
- [ ] Movie Description

Unsure if you can set the custom sort order with a metadata agent for collections, but if its possible it would be wanted.

**Comment by Darklyter** [comment]
It's currently not working with movies, just scenes.  Is that what you're talking about, the Stash movies?

**Comment by CuddleBear92** [comment]
Yes, you are already making collections with them. just missing the cover images and description if any.
